src/intertypes/python.jl

```diff
@@ -156,7 +154,7 @@ class InterTypeBase(BaseModel):
         return super().model_dump_json(*args, **kwargs, by_alias=True)
 """
 
-function generate_python_module(mod::InterTypeModule, path)
+function generate_module(mod::InterTypeModule, ::Type{ExportTarget{:python}}, path)
```

Comment:
Then when you use the type for multiple dispatch, it would be 

```julia
function generate_module(mod::InterTypeModule, ::Type{PythonTarget}, path)
...
end
```
and at the call site `generate_module(mod, PythonTarget, path)`